src/app/chat/messages/messages.actions.jsx

```diff
@@ -1,11 +1,12 @@
-import Reflux from 'reflux';
+import alt from '../../alt';
```

Comment:
https://github.com/itsthatguy/switchboard/pull/20/files#diff-041025232f4b808bd4a04bbcdd5ea106R1

https://github.com/goatslacker/alt#what-does-it-look-like